src/compat.h

```diff
@@ -254,6 +254,22 @@
 #define TUPLE_DESC_HAS_OIDS(desc) false
 #endif
 
+#if PG96
```

Comment:
What is this supposed to be used for? We no longer support PG96 and I don't see any of these macros being used in this PR.

Reply:
Merge problem. I first fixed the problem for 1.7.2 and then merged it upwards. Removed.